chasm/lib/activity/frontend.go

```diff
@@ -138,6 +143,88 @@ func (h *frontendHandler) GetActivityExecutionOutcome(
 	return resp.GetFrontendResponse(), err
 }
 
+// ListActivityExecutions lists activity executions matching the query in the request.
+func (h *frontendHandler) ListActivityExecutions(
+	ctx context.Context,
+	req *workflowservice.ListActivityExecutionsRequest,
+) (*workflowservice.ListActivityExecutionsResponse, error) {
+	namespaceID, err := h.namespaceRegistry.GetNamespaceID(namespace.Name(req.GetNamespace()))
+	if err != nil {
+		return nil, err
+	}
+	ctx = chasm.NewVisibilityManagerContext(ctx, h.visibilityManager)
```

Comment:
This is something that we expect to be done in an interceptor. Please coordinate with @awln-temporal @lina-temporal and @rodrigozhou on this.

Reply:
Is this blocking?

Reply:
OK I see the interceptor was added on `main`. I've rebased `standalone-activity` on `main` and rebased this PR on `standalone-activity` and got rid of this.